lobby.html

```diff
@@ -397,8 +423,14 @@
             drawLine(context, parsed.Data.FromX * scaleDownFactor, parsed.Data.FromY * scaleDownFactor, parsed.Data.ToX * scaleDownFactor, parsed.Data.ToY * scaleDownFactor, parsed.Data.Color, parsed.Data.LineWidth * scaleDownFactor);
         } else if (parsed.Type === "pixels") {
             parsed.Data.forEach(function(pixel) {
-                drawLine(context, pixel.FromX * scaleDownFactor, pixel.FromY * scaleDownFactor, pixel.ToX * scaleDownFactor, pixel.ToY * scaleDownFactor, pixel.Color, pixel.LineWidth * scaleDownFactor);
+                if (pixel.Type === "fill") {
```

Comment:
Do we really need "pixels"."fill" and "fill"?

Reply:
so this is a really ugly leftover that I quickly implemented to make it work and then forgot about.
Will deuglify this.